snapshotter/internal/http_address_resolver.go

```diff
@@ -44,6 +45,7 @@ func init() {
 	flag.IntVar(&port, "port", 10001, "service port for address resolver")
 	flag.StringVar(&containerdSockPath, "containerdSocket", "/run/firecracker-containerd/containerd.sock", "filepath to the containerd socket")
 	flag.IntVar(&remotePort, "remotePort", 10000, "the remote port on which the remote snapshotter is listening")
+	flag.IntVar(&metricsRemotePort, "metricsRemotePort", 10001, "the remote port on which the remote snapshotter metrics server is listening")
```

Comment:
I see that `10001` is being used as default for both address resolver port and for metrics remote port? intentional?

Reply:
The address resolver is running on the host while remote ports are used for services listening via vsock in VM, so there will not be a conflict. This internal resolver is used as an example and for integration testing. I can choose a different remote port for metrics since you're right - it does look fishy.